spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -1128,7 +1133,7 @@ object CometSparkSessionExtensions extends Logging {
     // Only consider converting leaf nodes to columnar currently, so that all the following
     // operators can have a chance to be converted to columnar.
     // TODO: consider converting other intermediate operators to columnar.
-    op.isInstanceOf[LeafExecNode] && !op.supportsColumnar && isSchemaSupported(op.schema) &&
```

Comment:
Could you update the comments here to cover this change in functionality?

Reply:
Added a little bit more, but still is just a little weird since it's technically still called `RowToColumnar`